dockerspawner/systemuserspawner.py

```diff
@@ -34,6 +34,21 @@ class SystemUserSpawner(DockerSpawner):
         ),
     )
 
+    image_homedir_propagation = Unicode(
+        "rprivate",
+        config=True,
+        help=dedent(
+            """
+            Mode for bind mount propagation. Possible values are
+            `rshared`,`shared`,`rprivate`, `private`, `rslave`, `slave`.
+            default is `rprivate`.
+            This is only interpreted by docker-py if patched  (see:
+            https://github.com/jannefleischer/docker-py/commit/786d55de465e72d0bb4b318272a2d020e43ff54a
```

Comment:
If it required a manual patch, I would not consider this patch appropriate for this project, but since it's merged, can you reference the required version of docker-py here? It's not clear to me what that is.